testsuite/mockserver/pytest_plugin.py

```diff
@@ -60,6 +60,11 @@ def pytest_addoption(parser):
             default=MOCKSERVER_SSL_DEFAULT_PORT,
         ),
     )
+    group.addoption(
```

Comment:
add info about new switch to documentation

Reply:
done